### PR TITLE
Updated all FAQs

### PR DESCRIPTION
**Pull request by matsduf**
## Purpose

FAQ 5 has the old mailing list address.

* Updated (corrected) mail address to mailing list.
* Added a comment that it is in fact a mailing list.

After the address I added "(moderated mailing list)" so that the send should know that. For Swedish I inserted a translated version, but for all other languages (da, es, fi, fr and nb) I inserted the same English text. Please add the correct wording for "your" language as a review comment to this PR.

2022-09-20: Turned the mail address into a mailto link, unless it already was.

## Context

On all, or most, installations of the GUI the wrong mail address is shown, and that results in an error.

## How to test this PR

Review the text.

### CODE REVIEW

**Comment by matsduf** [comment]
@huguei, please see this PR.

**Comment by huguei** [comment]
> @huguei, please see this PR.

Will do ASAP.

File: docs/FAQ/gui-faq-fr.md [hunk 1]
```diff
@@ -95,7 +95,8 @@ Mais, avec les technonologies évolutives, comme le DNS, ce qui est aujourd'hui
 un simple avertissement, peut devenir demain une faille de sécurité grave. Si
 vous considérez que nous avons fait une erreur de jugement pour qualifier une
 erreur, n'hésitez pas à nous en faire part en envoyant un Email à l'adresse
-zonemaster-devel@lists.iis.se avec le lien sur le test réalisé et un commentaire
+zonemaster-users@lists.iis.se (moderated mailing list) avec le lien sur le test
```
Comment (tgreenx) [inline docs/FAQ/gui-faq-fr.md:98]: (moderated mailing list) -> (liste email modérée)

File: docs/FAQ/gui-faq-es.md [hunk 2]
```diff
@@ -79,7 +79,8 @@ Pero tal como muchas cosas que evolucionan como el DNS, la situación es
 cambiante en el tiempo, y lo que es un aviso hoy puede ser un error
 mañana. Si realmente crees que hemos cometido un error en nuestro
 juicio, por favor no dudes en enviarnos un email a 
-zonemaster-devel@lists.iis.se con un enlace a tu prueba y una explicación
+zonemaster-users@lists.iis.se (moderated mailing list) con un enlace a tu prueba
```
Comment (huguei) [inline docs/FAQ/gui-faq-es.md:82]:
```suggestion
zonemaster-users@lists.iis.se (lista de correo moderada) con un enlace a tu prueba
```

Comment (matsduf) [reply docs/FAQ/gui-faq-es.md:82]: Fixed.

File: docs/FAQ/gui-faq-fr.md (hunk 1 above)
Comment (matsduf) [reply docs/FAQ/gui-faq-fr.md:98]: Fixed.

File: docs/FAQ/gui-faq-es.md (hunk 2 above)
Comment (repperille) [reply docs/FAQ/gui-faq-es.md]: Need a space in "sende en e-post til "

- sende en e-posttil zonemaster-users@lists.iis.se (moderated mailing list) med en lenke
+ sende en e-post til zonemaster-users@lists.iis.se (moderert e-postliste) med en lenke

**Comment by repperille** [comment]
Need a space in "sende en e-post til "
<pre>
- sende en e-posttil zonemaster-users@lists.iis.se (moderated mailing list) med en lenke
+ sende en e-post til zonemaster-users@lists.iis.se (moderert e-postliste) med en lenke
</pre>

File: docs/FAQ/gui-faq-nb.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ Det beror på hvilket test som gikk galt for domenet.
 #### 5. Hvordan avgjør Zonemaster hva som er rett eller feil? <a name="q5"></a>
 Det er viktig å understreke at ingen kan gi en endelig uttalelse om helsen til et domene. Zonemaster er ikke alltid fullstendig korrekt, og resultatene kan gi rom for tolkning. De som står bak testen understreker at de har gjort sitt aller beste for å utvikle en best mulig policy for vurdering av ulike feil før de blir presentert for deg som bruker verktøyet.
 
-Hvis du syns at et testresultat er feil så oppfordrer vi til å sende en e-post til zonemaster-devel@lists.iis.se med en lenke til testresultatet og forklar hvorfor du synes testen er feil.
+Hvis du syns at et testresultat er feil så oppfordrer vi til å sende en e-posttil zonemaster-users@lists.iis.se (moderated mailing list) med en lenke til testresultatet og forklar hvorfor du synes testen er feil.
```
Comment (repperille) [inline docs/FAQ/gui-faq-nb.md:53]: sende en e-post til zonemaster-users@lists.iis.se (moderert e-postliste) med en lenke

Comment (repperille) [inline docs/FAQ/gui-faq-nb.md:53]: Need a space in "sende en e-post til "
<pre>
- sende en e-posttil zonemaster-users@lists.iis.se (moderated mailing list) med en lenke
+ sende en e-post til zonemaster-users@lists.iis.se (moderert e-postliste) med en lenke
</pre>

Comment (matsduf) [reply docs/FAQ/gui-faq-nb.md]: @repperille, thank you. Fixed.

**Review by pnax — COMMENTED** [review]
Some translation provide a mailto link and others not. It might be nice to harmonize it and add the mailto link in all cases.

**Comment by matsduf** [comment]
> Some translation provide a mailto link and others not. It might be nice to harmonize it and add the mailto link in all cases.

Updated.

**Review by pnax — APPROVED** [review]

**Review by repperille — APPROVED** [review]

**Pull request merged**